actionpack/lib/abstract_controller/base.rb

```diff
@@ -22,6 +23,7 @@ class Base
     include ActiveSupport::Configurable
     extend ActiveSupport::DescendantsTracker
 
+    # Remove after ActionView::Template::Handlers::Erubis is removed
```

Comment:
Erubis adds a private Kernel#not_implemented method.  This undef_method was just to remove it.  However, if we don't require erubis, we don't need to undef the method, so this can be removed.

I'm not sure what the effects are if erubis is loaded later, though, as the method would end up being defined at that point.